### PR TITLE
TthCrstl update

### PR DESCRIPTION
**Pull request by NoSloppy**
Hierarchy made to be FAT32 friendly.

### CODE REVIEW

**Comment by profezzorn** [comment]
When you add files to these, you need to add something to the license file as well.
Also, I can't approve checking in files unless I know where the files came from.
If you created them yourself, then I need you to specify that and add something reasonable to the license file about it.

**Comment by NoSloppy** [comment]
Got it. This particular font has no adds, just rearranged. Should be good to go.

File: TthCrstl/smoothsw.ini [hunk 1]
```diff
@@ -1,16 +1,46 @@
...
+# Smoothswing volume multiplier (defaults to 3x normal volume)
+# Default is 3.0 (value between 1 and 5)
+MaxSwingVolume=3.0
+
+# What percent the hum sound will decrease as swing increases
+# Default is 75.0 (value between 1 and 100)
 MaximumHumDucking=75
-# Non-linear swing response, higher values makes it more non-linear.
-SwingSharpness=1.5
-# Degrees / second needed to register as a swing.
```
Comment (profezzorn) [inline TthCrstl/smoothsw.ini]: Thexter set this to 1.5, why are we changing it to 1.75?

File: TthCrstl/config.ini [hunk 2]
```diff
@@ -1 +1,76 @@
-humStart=260
\ No newline at end of file
+# Default config.ini
```
Comment (profezzorn) [inline TthCrstl/config.ini]: Why should this font have the default config.ini?

File: TthCrstl/smoothsw.ini (hunk 1 above)
Comment (NoSloppy) [reply TthCrstl/smoothsw.ini]: I didn't notice they weren't all the defaults. I'll set them back if the font was specifically different

File: TthCrstl/config.ini (hunk 2 above)
Comment (NoSloppy) [reply TthCrstl/config.ini]: If this is the creators intent for a humstart, so be it.

**Comment by NoSloppy** [comment]
I think these are all up to par now...?

**Comment by profezzorn** [comment]
Github says hum01.wav was modified, but I don't see any mention of any modifications.
Is that a mistake or not?
If it's not a mistake, you'll need to update the license file as well.

**Comment by NoSloppy** [comment]
redo

**Pull request closed without merging**